source/client-side-encryption/tests/README.rst

```diff
@@ -596,6 +606,97 @@ Data keys created with AWS KMS may specify a custom endpoint to contact (instead
 
    Expect this to fail with an exception with a message containing the string: "parse error"
 
+Azure and GCP custom endpoints
```

Comment:
Consider merging this test with the existing one since they are actually about the same thing. I did it here: https://github.com/DmitryLukyanov/mongo-csharp-driver/blob/4ea6f6553330ab58db0be9634dfe89cd1c79119c/tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/ClientEncryptionProseTests.cs#L556